### ISSUE TITLE
Property `sensitive` of resource `macosx_autologin` overwrites an existing method

### ISSUE THREAD

**Issue by brentleyjones** [dhoer/chef-macosx_autologin#2]
```
Deprecated features used!
  Property `sensitive` of resource `macosx_autologin` overwrites an existing method. Please use a different property name. This will raise an exception in Chef 13. at 1 location:
    - /tmp/kitchen/cache/cookbooks/macosx_autologin/resources/default.rb:8:in `class_from_file'
   See https://docs.chef.io/deprecations_property_name_collision.html for further details.
```

**Comment by dhoer** [comment]
Is `confidential` an existing method?  If not, would you mind submitting a PR replacing `sensitive` with this instead?

**Comment by dhoer** [comment]
Released v5.0.0
